devise.gemspec

```diff
@@ -14,6 +14,13 @@ Gem::Specification.new do |s|
   s.homepage    = "https://github.com/heartcombo/devise"
   s.description = "Flexible authentication solution for Rails with Warden"
   s.authors     = ['José Valim', 'Carlos Antônio']
+  s.metadata    = {
+    "homepage_uri"     => "https://github.com/heartcombo/devise",
+    "changelog_uri"    => "https://github.com/heartcombo/devise/blob/master/CHANGELOG.md",
+    "source_code_uri"  => "https://github.com/heartcombo/devise/issues",
+    "wiki_uri"         => "https://github.com/heartcombo/devise/wiki",
+    "mailing_list_uri" => "http://stackoverflow.com/questions/tagged/devise"
```

Comment:
We actually have a mailing list under heartcombo: https://groups.google.com/g/heartcombo... I'm not sure I want to point people to there though as I have been thinking about killing it, but maybe it's a better option to start. I'm also not sure about pointing to stackoverflow here, but I guess that's where we point people from the readme anyway... just seems a little odd to have it here for some reason 😅

Reply:
Ah, yah I was just grabbing it from the readme but definitely up to the core contributors. I think the Google Groups mailing list must be private as it redirects me to https://groups.google.com/access-error?continue=https://groups.google.com/g/heartcombo. We could of course just remove it for now, I was just trying to enhance the metadata with [what's available](https://guides.rubygems.org/specification-reference/#metadata).